### ISSUE TITLE
Simple Auth bugs

### ISSUE THREAD

**Issue by astersx** [PocketMine/SimpleAuth#94]
In console it show many warning messages,  Illegal ...... etc.. 
If i /reload server simple auth become red and no work.
Accounts can be hacked,  and i used a veeery long and strong password (i deoped my username and now my op name secret).

Do you plan to update Simple Auth????

**Comment by PEMapModder** [comment]
After you typed in the password and the account got saved, nobody including SimpleAuth itself (except yourself) knows the password because it is encrypted. Are you sure that others got the password, or is it somehow you misconfigured the plugin and let people use the account without logging in?

**Comment by astersx** [comment]
Nono,  i just found some blocks in my spawn was missing,  and my spawn is protected with iprotect and economyland,  so i dont think someone can grief it!!!
Anyway will this plugin be uptaded? If i /reload server it is red and dont work :(!!! 

And plugin is from minecraft oficial plugins page!!!

**Comment by PEMapModder** [comment]
Please don't say "illegal something". Full message please. That's the important thing...

**Comment by astersx** [comment]
[10:46:48] [Server thread/CRITICAL]: "Could not pass event 'pocketmine\event\player\PlayerQuitEvent' to 'SimpleAuth v1.7.1': Illegal offset type in unset on SimpleAuth\EventListener
[10:46:48] [Server thread/WARNING]: RuntimeException: "Illegal offset type in unset" (E_WARNING) in "/SimpleAuth_v1.7.1.phar/src/SimpleAuth/task/ShowMessageTask" at line 46

**Comment by astersx** [comment]
Its spamming my console

**Comment by astersx** [comment]
I dont know if someone care about this ..  But players can craft items even if they didnt log in....

**Comment by PEMapModder** [comment]
Is that the only message? Are there any other messages that do not contain PlayerQuitEvent?

**Comment by astersx** [comment]
Playerjoin too

**Comment by PEMapModder** [comment]
Message?

**Comment by astersx** [comment]
[12:30:49] [Server thread/CRITICAL]: "Could not pass event 'pocketmine\event\player\PlayerJoinEvent' to 'SimpleAuth v1.7.1': Illegal offset type on SimpleAuth\EventListener
[12:30:49] [Server thread/WARNING]: RuntimeException: "Illegal offset type" (E_WARNING) in "/SimpleAuth_v1.7.1.phar/src/SimpleAuth/task/ShowMessageTask" at line 42

**Comment by dterweij** [comment]
Just installed this plugin. It does not work. 

RuntimeException: "Illegal offset type" (E_WARNING) in "/SimpleAuth_v1.7.1.phar/src/SimpleAuth/task/ShowMessageTask" at line 42

**Comment by astersx** [comment]
Yes is broken,  it work with simpleauth,  but dont /reload server,  if u reload,  /stop fast then reopen.
I have no other options but to use this plugin else i need restart all accounts and all can lose accounts, so im doomed becouse shoghip dont care.
But as long there is a way to work im ok!!!  :

**Comment by Kainzo** [comment]
[21:42:40] [Server thread/CRITICAL]: "Could not pass event 'pocketmine\event\player\PlayerJoinEvent' to 'SimpleAuth v1.7.1': Illegal offset type on SimpleAuth\EventListener
[21:42:40] [Server thread/WARNING]: RuntimeException: "Illegal offset type" (E_WARNING) in "/SimpleAuth_v1.7.1.phar/src/SimpleAuth/task/ShowMessageTask" at line 42
[21:42:40] [Server thread/INFO]: Kainzo joined the game
[21:43:12] [Server thread/CRITICAL]: Unhandled exception executing command 'register 076871' in register: Illegal offset type in unset
[21:43:12] [Server thread/WARNING]: RuntimeException: "Illegal offset type in unset" (E_WARNING) in "/SimpleAuth_v1.7.1.phar/src/SimpleAuth/task/ShowMessageTask" at line 46

im seeing this too.

**Comment by Aericio** [comment]
Im seeing that too. Use SimpleAuthHelper and the bug is fixed.

**Comment by PEMapModder** [comment]
@Aericio this is not a forum.

**Comment by Kainzo** [comment]
@PEMapModder - but it works the same as a forum, so who cares?

**Comment by PEMapModder** [comment]
This does **not** work like a forum. This is an issue tracker.
